content/en/docs/actions.md

```diff
@@ -0,0 +1,117 @@
+
+#### Scenario
+
+- Preemption between jobs in the same queue: Multiple departments in a company share a cluster, and each department can be mapped into a Queue. Resources of different departments cannot be preempted from each other. This mechanism can well guarantee the isolation of resources of departments..In complex scheduling scenarios, basic resources (CPUs, disks, GPUs, memory, network bandwidth) are allocated based on services: In computing-intensive scenarios, such as AI and high-performance scientific computing, queues require more computing resources, such as CPUs, GPUs, and memory. Big data scenarios, such as the Spark framework, have high requirements on disks. Different queues share resources. If AI jobs preempts all CPU resources, jobs in queues of other scenarios will starve. Therefore, the queue-based resource allocation is used to ensure service running.
+- Preemption between tasks in the same job: Usually, there can be multiple tasks in the same Job. For example, in complex AI application scenarios, a parameter server and multiple workers need to be set inside the TF-job, and preemption between multiple workers is supported by preemption within such scenarios.
+
+
+
+### Reclaim
+
+#### Overview
+
+Reclaim action is a scheduling process for a recovery step. A queue is used in the volcano to allocate resources proportionally. When a queue is added or removed from the cluster, Reclaim will take charge of recycling and redistributing resources to the remaining queues.
+
+#### Scenario
+
+In the mixed batch computing business scenario (kubeflow/flink/tensorflow, etc.), different business in queue allocation the cluster resources, at this time, if a new business, Reclaim for new queue allocation ratio of cluster resources. However, once a new queue is allocated quotas, it does not indicate that pods under the queue can be properly scheduled, because the quota allocated to the queue at this point is only the upper limit for the use of cluster resources, and does not guarantee the use of the cluster. If the old business happens to be using up all the cluster resources, the new business will be "busy waiting." At this point, you needed reclaim action to rebalance resources between different queues. Reclaim action tried to evict pods from queues whose resource usage had exceeded the quota, and to allocate resources to queues whose resource usage had not been met.
+
+Reclaim action tried to evict Pods from queues whose resource usage had exceeded the quota, and to allocate resources to queues whose resource usage had not been met. Reclaim action ensured the flexibility of cluster resource allocation and prevented the shock phenomenon of mutual expulsion among queues.
+
+
+
+### Elect
```

Comment:
The action has been deprecated from v1.2.  Please remove this action or mark it as deprecated.